src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
Please specify an access modifier for the Ball class. If you want to ensure that this class is accessible throughout your application, you should declare it as `public`.